src/Console/Infrastructure/Command/ListModulesCommand.php

```diff
@@ -25,55 +26,62 @@ protected function configure(): void
         $this->setName('list:modules')
             ->setDescription('Render all modules found')
             ->addArgument('filter', InputArgument::OPTIONAL, 'Any filter to simplify the output')
-            ->addOption('simple', 's', InputOption::VALUE_NONE, 'Display just the module names');
+            ->addOption('detailed', 'd', InputOption::VALUE_NONE, 'Display all the modules in detail');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $filter = (string)$input->getArgument('filter');
 
-        $listOfModules = $this->generateListOfModules(
-            (bool)$input->getOption('simple'),
+        $this->generateListOfModules(
+            $output,
```

Comment:
What about setting the `output` as private property of the command (eg: in the like 34), so you can access it any level below (on any private func) instead of passing it as an extra argument to 2 or 3 nested methods?